### ISSUE TITLE
missing depends: traverse

### ISSUE THREAD

**Issue by uwabami** [kijimaD/textlint-plugin-org#100]
```
% textlint test.org

✖ Error
Cannot find module 'traverse'
Require stack:
- ~/nodejs/lib/node_modules/textlint-plugin-org/lib/org-to-ast.js
- ~/nodejs/lib/node_modules/textlint-plugin-org/lib/OrgProcessor.js
- ~/nodejs/lib/node_modules/textlint-plugin-org/lib/index.js
- ~/nodejs/lib/node_modules/textlint/lib/src/engine/textlint-module-loader.js
- ~/nodejs/lib/node_modules/textlint/lib/src/engine/textlint-engine-core.js
- ~/nodejs/lib/node_modules/textlint/lib/src/textlint-engine.js
- ~/nodejs/lib/node_modules/textlint/lib/src/cli.js
- ~/nodejs/lib/node_modules/textlint/bin/textlint.js

✖ Stack trace
Error: Cannot find module 'traverse'
Require stack:
- ~/nodejs/lib/node_modules/textlint-plugin-org/lib/org-to-ast.js
- ~/nodejs/lib/node_modules/textlint-plugin-org/lib/OrgProcessor.js
- ~/nodejs/lib/node_modules/textlint-plugin-org/lib/index.js
- ~/nodejs/lib/node_modules/textlint/lib/src/engine/textlint-module-loader.js
- ~/nodejs/lib/node_modules/textlint/lib/src/engine/textlint-engine-core.js
- ~/nodejs/lib/node_modules/textlint/lib/src/textlint-engine.js
- ~/nodejs/lib/node_modules/textlint/lib/src/cli.js
- ~/nodejs/lib/node_modules/textlint/bin/textlint.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (~/node_modules/textlint-plugin-org/lib/org-to-ast.js:8:34)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
```

Best Wishes,

**Comment by kijimad** [comment]
@uwabami 
Sorry for the late reply. This problem probably fixed on #295 recently.
How about can you use latest version textlint-plugin-org(v0.3.4)?

**Comment by uwabami** [comment]
Sorry late reply. confirmed. thx!
